src/data/bounties.json

```diff
@@ -64,7 +64,7 @@
     },
     {
       "repository": "thenewboston-js",
-      "title": "[Task] Create `Bank` Documentation",
```

Comment:
no need to change the content of the `.json` files

Reply:
done!